src/sd.c

```diff
@@ -107,7 +108,14 @@ static bool _mount(void)
     sd_mmc_resume_clock();
 #endif
     memset(&fs, 0, sizeof(FATFS));
-    if (f_mount(&fs, "SD", 1) == FR_INVALID_DRIVE) {
+    int res = f_mount(&fs, "", 1);
+    if (res == FR_DISK_ERR) {
+#ifndef TESTING
+        sd_mmc_start();
+#endif
+        res = f_mount(&fs, "", 1);
+    }
+    if (res == FR_INVALID_DRIVE) {
```

Comment:
what if res is some other error?

Reply:
Yeah, that check could be `res != FR_OK`. It would change the behavior of mount to return false in more cases. But that is probably for the good. Right now we are ignoring errors.